event.go

```diff
@@ -69,7 +69,7 @@ func jobStatisticsFromInternal(stats *jobstats.JobStatistics) *JobStatistics {
 
 // The maximum size of the subscribe channel. Events that would overflow it will
 // be dropped.
-const subscribeChanSize = 100
+const subscribeChanSize = 50_000
```

Comment:
Had to substantially increase this number of the benchmarker since it's now possible to insert 100 jobs to the channel almost instantaneous and even a fast channel listener can lose some.

Reply:
I was wondering earlier if this would be an issue with the way you were now measuring. It’s unlikely someone would hit this in practice but I suppose it’s still a bit of a wart with the current fixed size buffered channel for subscriptions.